src/utils/InternalException.ts

```diff
@@ -0,0 +1,3 @@
+export class InternalException extends Error {
+    public isAzureFunctionsInternalException = true;
+}
```

Comment:
Nit: missing newline at the end of this file.

Reply:
ty!! also reminds me that we need to enable ts-lint on this repo :O thanks!!